### ISSUE TITLE
Error with Deploy To Azure

### ISSUE THREAD

**Issue by NicolasDorier** [outsideris/slack-invite-automation#79]
```
Deployment Error
InvalidRequestContent: The request content was invalid and could not be deserialized: 'Could not find member 'default' on object of type 'TemplateInputParameter'. Path 'properties.template.parameters.siteLocation.default', line 15, position 21.'.
```

Any idea ?

**Comment by xeniacx** [comment]
I have exactly the same issue:

![deploy-erro](https://cloud.githubusercontent.com/assets/574301/24556462/18298976-163d-11e7-937f-a8eb6755e3ef.jpg)

**Comment by ChrisAnn** [comment]
Same.  Any idea of a fix?

**Comment by ghost** [comment]
I got the same error. :\

**Comment by NicolasDorier** [comment]
@outsideris still maintaining?

**Comment by outsideris** [comment]
@NicolasDorier Yes. but I've not been use Azure ever.

**Comment by CKGrafico** [comment]
nothing yet?

**Comment by outsideris** [comment]
fixed in 1e25ee648a89bc8ee7b05db3510e7ac55dfad1c1
